partials/tabs/faq.html

```diff
@@ -14,6 +14,8 @@
 <script>
 $(document).ready(function() {
   $('.collapsible').collapsible();
+    $('ul.tabs').tabs();
+
```

Comment:
Is there a reason for leaving this line?

Reply:
If Travis is failing coz there is no newline here, why isn't it failing https://github.com/coala/projects/pull/697/files#diff-b5dc5a5e22c39da3a9de7f98539bde5eR78?